developing/how-to-update-openstack-upgrade-path.md

```diff
@@ -46,6 +46,57 @@ the debian package `bind9`, and we can use `rmadison bind9` to find all the
 versions for charm `designate-bind`. Similarly, for charm `gnocchi`, the
 versions can be found using `cmadison gnocchi`.
 
+The list of package need to be updated are:
+
+```shell
+# Install rmadison
+apt install devscripts
+# Install cmadison
+snap install cmadison
+
+# Find the upper version for the specific releases
+rmadison gnocchi
+rmadison bind9
+rmadison rabbitmq-server
+
+# For example, the upper version for rabbitmq-server in Jammy/2023.2 (Bobcat) is
```

Comment:
How do you map from openstack release to ubuntu release? Especially for releases like yoga that could be on focal or jammy.

Reply:
There's no particular rule here, as long as it's a good educated guess, like the following sentence mentioned